### PR TITLE
feat: add inbox icon

### PR DESCRIPTION
**Pull request by tiagoporto**
[TDESKAPP-558]

### Description

Add inbox icon

### How to test

Open `http://localhost:9001/?path=/story/components-icons--icons` and search for inbox icon.

### Screenshots

image

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[TDESKAPP-558]: https://toptal-core.atlassian.net/browse/TDESKAPP-558?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/angry-countries-wave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': minor
+---
+
+Add inbox icon
```
Comment (augustobmoura) [inline .changeset/angry-countries-wave.md:5]: Small change here, we need to follow a [changeset template](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md#summary-of-the-change-changelog-message)

**Comment by augustobmoura** [comment]
I checked the Happo screenshot and it's looking fine.
There's only a small change that we need to fix on the changeset and we good to go!

Thanks for your contribution @tiagoporto! :+1: 
We appreciate it!

File: .changeset/angry-countries-wave.md (hunk 1 above)
Comment (tiagoporto) [reply .changeset/angry-countries-wave.md]: Fixed

**Review by OndrejTuma — APPROVED** [review]
Checked the code and happo screenshot, I see to problems. Good job 💪

**Review by LashaJini — APPROVED** [review]
Nice, checked locally and in the demo. nit: In the description I would replace `localhost` with demo generated in `https://picasso.toptal.net/TDESKAPP-558-inbox-icon/`, so something like: https://picasso.toptal.net/TDESKAPP-558-inbox-icon/?path=/story/components-icons--icons. Thank you for contribution :bow:

**Pull request merged**